src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -227,8 +280,8 @@ public override string ToString()
             return string.Format(CultureInfo.CurrentCulture, $"{{MessageId:{this.MessageId}}}");
         }
 
-        /// <summary>Clones a message, so that it is possible to send a clone of a message as a new message.</summary>
-        /// <returns>The <see cref="Message" /> that contains the cloned message.</returns>
+        /// <summary>Clones a message, so that it is possible to send a clone of an already received message as a new message.</summary>
```

Comment:
System properties are [not cloned](https://github.com/Azure/azure-service-bus-dotnet/blob/468d02fd16a314ba967662fe382e3457c5b1e1f7/src/Microsoft.Azure.ServiceBus/Message.cs#L245). Should that be added as a remark?

Reply:
Since they're set by the system that appears to be implied for a fresh message.